src/components/AChat/AChatForm.vue

```diff
@@ -232,6 +246,30 @@ export default {
     },
     focus() {
       this.$refs.messageTextarea.focus()
+    },
+    selectCommand(direction) {
+      if (!this.message) {
+        this.botCommandSelectionMode = true
+      }
+      if (this.botCommandSelectionMode) {
+        const commands = this.$store.getters['botCommands/getCommandsHistory'](this.partnerId)
+        const maxIndex = commands.length > 0 ? commands.length - 1 : 0
+        if (this.botCommandIndex === null) {
+          if (direction === 'ArrowUp') {
+            this.botCommandIndex = maxIndex
+          }
+        } else {
+          if (direction === 'ArrowUp') {
+            if (this.botCommandIndex > 0) {
+              this.botCommandIndex--
+            }
+          } else if (this.botCommandIndex < maxIndex) {
+            this.botCommandIndex++
+          }
+        }
+
+        this.message = commands[this.botCommandIndex] || ''
+      }
```

Comment:
Avoid using deep nested if/else conditions. Instead try using early return. This will improve the code readability.

E.g.:

```js
if (!this.botCommandSelectionMode) {
   return;
}

const commands = this.$store.getters['botCommands/getCommandsHistory'](this.partnerId)
// ...
```